### DISCUSSION TITLE
Outdated version constraint for `statamic/cms`?

### DISCUSSION THREAD

**Discussion by adevade** [spatie/statamic-responsive-images#220] [Q&A] [answered]
This should not match 3.4.* of Statamic, right? I'm not sure why it's working for me... I'm running 3.4.9, and the latest version of this addon.

https://github.com/spatie/statamic-responsive-images/blob/f70dfc43a16ae6d987b091c3685f3e925aa7faa2/composer.json#L27-L29

Semver checker: https://jubianchi.github.io/semver-check/#/~3.3/3.4.9

**Comment by adevade** [comment, marked as answer]
Well, now I actually read the text on the website I linked to 😅

> Composer handles tilde constraint differently. Your constraint will translate to >=3.3.0 <4.0.0.

**Reply by ncla** [reply]
Yeah this is a pretty uhh funny issue that I ran into too. I think with Statamic v4 we should transition to ^ operator, as v4 now will use semver. :)
